recipes/default.rb

```diff
@@ -15,3 +15,166 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+
+node.default['osl-apache']['listen'] = %w(80 443)
+node.default['osl-apache']['worker_mem'] = 215
+
+node.default['osl-postfix']['main']['home_mailbox'] = 'Mail/'
+node.default['osl-postfix']['main']['mailbox_command'] = '/usr/bin/procmail'
+node.default['osl-postfix']['main']['mailbox_size_limit'] = '0'
+node.default['osl-postfix']['main']['message_size_limit'] = '102400000'
+node.default['osl-postfix']['main']['transport_maps'] = 'hash:/etc/postfix/transport'
+
+node.default['postfix']['access']['140.211.166.133'] = 'OK' # smtp2.osuosl.org
+node.default['postfix']['access']['140.211.166.136'] = 'OK' # smtp3.osuosl.org
+node.default['postfix']['access']['140.211.166.137'] = 'OK' # smtp4.osuosl.org
+node.default['postfix']['access']['140.211.166.138'] = 'OK' # smtp1.osuosl.org
+
+include_recipe 'osl-apache'
+include_recipe 'osl-apache::mod_ssl'
+include_recipe 'osl-apache::mod_perl'
+include_recipe 'osl-mysql::client'
+include_recipe 'yum-osuosl'
+include_recipe 'perl'
+
+package %w(request-tracker mutt procmail)
+
+# Root Account
+template '/root/.rtrc' do
+  source 'rt/rtrc.erb'
+  cookbook 'osl-rt'
```

Comment:
This isn't needed when we're doing it from a recipe. Please remove globally. We only need this inside of a resource.

Reply:
Unfortunately, it would be preferable to have the RT config set up, as we do run RT commands that make calls to the API. This is done around line `134`, with setting up the queues.

Unless, are you saying that we should instead have the config options in the command itself?

Reply:
I'm preferring to removing the `cookbook` parameters. Sorry for the confusion.